elasticsearch/src/main/java/org/hibernate/search/elasticsearch/analyzer/impl/NamedElasticsearchAnalyzerReference.java

```diff
@@ -35,6 +35,12 @@
 
 	private static final Log LOG = LoggerFactory.make( MethodHandles.lookup() );
 
+	private static final ElasticsearchAnalysisDefinitionRegistryPopulator NOOP_DEFINITION_REGISTRY_POPULATOR = new ElasticsearchAnalysisDefinitionRegistryPopulator() {
+		@Override
+		public void populate(ElasticsearchAnalysisDefinitionRegistry registry) {
+		}
+	};
```

Comment:
You really don't like lambdas, do you? :) You could have used one here.

Reply:
You know, I'm an old man now :). Fixed.